scripts/da_chgres_cold2fv3.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+from netCDF4 import Dataset
+import remap_dwinds
+import remap_scalar
+import chgres_winds   # might need to rename in the future
+import sys
+
+print("Reading in NETCDF4 Files... ", end="\r")
+warm = str(sys.argv[1])
+cold = str(sys.argv[2])
+grid = str(sys.argv[3])
+akbk = str(sys.argv[4])
+akbkcold = str(sys.argv[5])
+orog = str(sys.argv[6])
+
+warmnc = Dataset(warm)
+coldnc = Dataset(cold, mode="a")
+akbknc = Dataset(akbk)
+gridnc = Dataset(grid)
+akbkcoldnc = Dataset(akbkcold)
+orognc = Dataset(orog)
+print("Reading in NETCDF4 Files... Done.")
+
+ColdStartWinds = True
+VertRemapScalar = True
+VertRemapWinds = True
+WriteData = True
+
+# STEP 1. ROTATE THE WINDS FROM CHGRES
+if ColdStartWinds:
+    print("Starting ColdStartWinds.... ", end="\r")
+    # Data from warm restarts
+    u = np.float64(warmnc["u"][0, :, :, :])
+    v = np.float64(warmnc["v"][0, :, :, :])
+    nlev = np.shape(u)[0]  # 127,z
+    nlat = np.shape(u)[1]  # 769,y
+    nlon = np.shape(u)[2]  # 768,x
+
+    km = nlev
+    nlev = coldnc.createDimension("nlev", nlev)  # 127
+
+    # Data from cold chgres
+    u_s = np.float64(coldnc["u_s"][:, :, :])   # (128, 769, 768)
+    v_s = np.float64(coldnc["v_s"][:, :, :])   # (128, 769, 768)
+    u_w = np.float64(coldnc["u_w"][:, :, :])   # (128, 768, 769)
+    v_w = np.float64(coldnc["v_w"][:, :, :])   # (128, 768, 769)
+
+    # grid data - usually has 2x as many grid points, so we need every other value.
+    gridx = np.float64(gridnc["x"][0:-1:2, 0:-1:2])
```

Comment:
do you know why this has 2x has many points? is this the super grid?

Reply:
No, I'm not sure why it has 2x as many. For example:
```
ncdump -h /lfs/h2/emc/global/noscrub/emc.global/FIX/fix/orog/20220805/C768/C768_grid.tile1.nc
netcdf C768_grid.tile1 {
dimensions:
        string = 255 ;
        nx = 1536 ;
        ny = 1536 ;
        nxp = 1537 ;
        nyp = 1537 ;
```
I just know that this part is related only to the wind rotation part which I was able to reproduce within rounding error.